src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Use nouns for things. Consider more descriptive names such as 'deadlineDate' instead of 'by'.

Reply:
Will do, thanks!